docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -1,5 +1,5 @@
 include::./attributes.adoc[]
-= {project-name} - Building a Native Image
+= {project-name} - Building a Native Executable
```

Comment:
Should we change the guide file name too?

I'm pretty sure we are also using native image in other guides, don't we? Should we change it everywhere? Or did you already check?

Reply:
It's been reported on this one, but better check, I will let you know.

Changing the file name means changing the links, as the links have been sent around a few times already, better not changing it already. I plan to change it during the release process.